test/user/signin.test.js

```diff
@@ -0,0 +1,47 @@
+const request = require('supertest');
+const app = require('../../app');
+const { AUTHENTICATION_ERROR } = require('../../app/errors');
+
+describe('Sign in: POST /users/sessions', () => {
+  const credentials = {
+    email: 'david@wolox.com.co',
+    password: 'ABC12345678'
+  };
+
+  const user = {
+    name: 'David',
+    last_name: 'Sandoval',
+    ...credentials
+  };
+
+  beforeEach(async () => {
+    await request(app)
+      .post('/users')
+      .send(user);
+  });
+
+  test('it should sign in successfully', async () => {
+    const data = await request(app)
+      .post('/users/sessions')
+      .send(credentials);
+    expect(data.statusCode).toEqual(200);
+    expect(data.body).toHaveProperty('token');
+  });
+
+  test('it should validate that the credentials are valid', async () => {
```

Comment:
The message should be something like: `it should not validate the user with invalid credentials` what do you think?